services/core-api/app/api/mines/tailings/models/tailings.py

```diff
@@ -136,7 +158,8 @@ def create(cls,
             storage_location=storage_location,
             facility_type=facility_type,
             tailings_storage_facility_type=tailings_storage_facility_type,
-            mines_act_permit_no=mines_act_permit_no
+            mines_act_permit_no=mines_act_permit_no,
+            is_draft=is_feature_enabled(Feature.TSF_V2)
```

Comment:
I'm pretty sure this line won't be necessary when we remove the feature flag- the db column default should take over correctly